tutorials/ui/gui_skinning.rst

```diff
@@ -48,32 +23,125 @@ user interfaces.
    Don't use the custom theme option in the Project Settings, as there
    are known bugs with theme propagation. Instead, apply your theme to the
    root Control node's Theme property. It will propagate to instanced scenes
-   automatically. To get correct theming in the editor for instanced scenes,
+   automatically. To get correct themeing in the editor for instanced scenes,
    you can apply the theme resource to the instanced scene's root node as well.
 
-Theme options
--------------
+Creating a theme
+----------------
 
-Each kind of option in a theme can be:
+Theme's can be created from any control node. Select a control node in the scene
+hierarchy, then in the inspector go to the theme property. From there you can
+select **New Theme**.
 
--  **An integer constant**: A single numerical constant. Generally used
-   to define spacing between components or alignment.
--  **A Color**: A single color, with or without transparency. Colors are
+.. image:: img/new_theme.png
+
+This will create an empty theme and open up the theme editor.
+
+.. image:: img/theme_editor.png
+
+In the theme editor you can customize everything about a theme except for
+the default font the theme uses. That can only be customized in the inspector under
+the selected theme.
+
+.. image:: img/default_font.png
+
+Theme items
+-----------
+
+In the theme editor, next to the default preview window, is where you make changes
+to your theme. Clicking the plus button opens the add item type menu.
```

Comment:
> Clicking the plus button opens the add item type menu.

This is hard to read, should add some formatting about the item menu text to distinguish it from the rest of the sentence.